test/tests/functional/pbs_dup_acc_log_for_resv.py

```diff
@@ -58,19 +58,16 @@ def check_log_resv_time(self, accounting_log_msg, start_tup,
         This is a helper function to reterive start and end time
         of standing rervation from accounting log.
         """
-        find_start = m[start_tup][end_tup].find("start=")
-        find_end = m[start_tup][end_tup].find("end=")
-        find_duration = m[start_tup][end_tup].find("duration=")
+        loglist_value = accounting_log_msg[start_tup][end_tup]
+        log_value = loglist_value.split(" ")
         if check_status == "start":
-            time_str = m[start_tup][end_tup][
-                find_start:find_start + (find_end - find_start - 1)]
-            new_time_str = time_str.replace("\'", "")
-            actual_reservation_time = new_time_str.split("=")[1]
+            for i in log_value:
+                if i.startswith("start="):
+                    actual_reservation_time = i.replace("start=", "")
```

Comment:
Even my way works too, I agreed the way you suggested is more pythony ,so  implemented the above code as you  suggested  .